packages/nimble-components/src/select/index.ts

```diff
@@ -1,17 +1,28 @@
-import { attr, html, observable } from '@microsoft/fast-element';
+import {
```

Comment:
Having steeped in this a bit I found myself needing to pull up FoundationSelect side-by-side to follow the code because we are overriding so much of the behavior.

Are we getting much value out of not forking the foundation implementation? I can't help but feel we have tipped the scale on this control such that if the FoundationSelect did have significant bug fixes or feature additions we probably won't be accelerated by leveraging those changes but get stuck trying to integrate against them.

I think we would still leverage the FormAssociatedSelect base class and the DelegatesARIASelect mixins. Sorry for coming at it with that kind of discussion so late but if we do go with owning the implementation I wouldn't expect it to be a huge amount of work to inline and merge the changes. But if it is that would be an argument against.

Reply:
At the moment I'm wary of moving forward with that effort, because I think that's an unknown amount of work. We aren't just inheriting from the FAST Select, but also the FAST Listbox, by virtue of the FAST Select, and we _are_ leveraging some of the work each class does for various situations.

That said, I don't disagree with the sentiment and think we should revisit this.

Reply:
Too be clear, I'm not saying to online the full tree, just the contents of FoundationSelect, above FoundationSelect the hierarchy doesn't change. Ie still inherit from FormAssociatedSelect, etc.

That's where my thought it would not be too hard comes from, is there something In missing there?

Reply:
Ah...I'll take a look that that.

Reply:
I'll hold off on re-reviewing until this update.

Reply:
Done.